lib/pool.js

```diff
@@ -244,10 +244,10 @@ class Pool {
   async _makeConnection() {
     this.counters.makeCall += 1;
     if (
-      Object.keys(this.all_connections).length > this.config.maxLimit ||
-      this.counters.closeCall != this.counters.closeExit
+      (process.env.PINO_LOG_ENABLED && (process.env.PINO_LOG_ENABLED == 'true' || process.env.PINO_LOG_ENABLED == '1')) && (Object.keys(this.all_connections).length > this.config.maxLimit ||
+      this.counters.closeCall != this.counters.closeExit)
     ) {
-      logger.level = "info";
+      logger.evel = "info";
```

Comment:
This looks like a typo.

Reply:
You are right, slipped through since it is dynamic properties, so no compiler checks for this.  Made a commit just now to fix.  Thanks for noticing.